src/gui/iconjob.cpp

```diff
@@ -32,6 +32,7 @@ void IconJob::finished(QNetworkReply *reply)
         return;
 
     reply->deleteLater();
+    deleteLater();
```

Comment:
Better change to this->deleteLater() (just consistence and clarity with respect to the general codebase style)

Reply:
No problem :)